package.json

```diff
@@ -108,8 +108,6 @@
     "btoa": "^1.1.2",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
-    "eslint": "^5.3.0",
```

Comment:
Shouldn't we still have `eslint` as a dependency as we are using it on `npm run lint`?

Reply:
I've added a commit comment

> use eslint from superdesk-code-style

Eslint is already a dependency in `superdesk-code-style` repo, which is a dependency for this one, so eslint is already there. Not sure if this is a good practice, just leaving it as it were before this PR.

Reply:
oh okay, I thought we had eslint before